src/components/settings-menu/SettingsSection.js

```diff
@@ -44,6 +44,7 @@ const { RainbowRequestReview, RNReview } = NativeModules;
 
 export const SettingsExternalURLs = {
   rainbowHomepage: 'https://rainbow.me',
+  rainbowLearn: 'https://rainbow.me/learn',
```

Comment:
Also FYI this link doesn't work

Reply:
it will pre release, tweaking a few things